src/components/Cards/ExpertList.js

```diff
@@ -2,28 +2,35 @@ import Image from 'next/image';
 import Link from 'next/link';
 import styled from 'styled-components';
 
+import Footer from '../Footer';
+
 import {data} from './data/data';
 
 const Ul = styled.ul`
 	padding: 0;
+	font-size: 14px;
+	font-weight: 300;
 	list-style: none;
 `;
 
 const Anchor = styled.a`
-	color: black;
+	color: #767676;
 	text-decoration: none;
 `;
 
 const Card = styled.div`
 	display: flex;
 	flex-direction: row;
-	margin: 0 50px 30px 50px;
+	align-items: center;
```

Comment:
`flex-direction: row` is the default value for flex-direction and you don't need to declare this.